eth/handler.go

```diff
@@ -321,23 +321,23 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 	}
 	defer msg.Discard()
 
```

Comment:
@sid1602 Some messages here are logged as size=1. Find out when that happens (especially for `BlockBodiesMsg`, `NodeDataMsg`, and `Receipts`.) Maybe easier to debug at `SendEthSubproto` at `p2p/message.go:116`.